params/config.go

```diff
@@ -286,15 +286,16 @@ func NewNodeConfigWithDefaults(dataDir, fleet string, networkID uint64) (*NodeCo
 		nodeConfig.WhisperConfig.DataDir = path.Join(dataDir, "wnode")
 	}
 
-	statusConfigJSON, err := static.Asset(fmt.Sprintf("../config/cli/fleet-%s.json", fleet))
-	if err == nil {
-		err = LoadConfigFromJSON(string(statusConfigJSON), nodeConfig)
-	}
-	if err != nil {
-		return nil, fmt.Errorf("Default config could not be loaded: %s", err)
+	if fleet != FleetUndefined {
+		statusConfigJSON, err := static.Asset(fmt.Sprintf("../config/cli/fleet-%s.json", fleet))
+		if err == nil {
+			err = LoadConfigFromJSON(string(statusConfigJSON), nodeConfig)
+		}
+		if err != nil {
+			return nil, fmt.Errorf("Default config could not be loaded: %s", err)
```

Comment:
Errors should start with lower case.

Reply:
oh, missed that. Will fix it and merge, if no one else says anything against.

Reply:
wonder if there's a linter extension to catch that